src/modules/Dashboard/features/Participant/Assignments/ActivityListItem/ActivityListItem.tsx

```diff
@@ -25,6 +25,7 @@ export const ActivityListItem = ({
       </StyledActivityThumbnailContainer>
       <StyledActivityName>{name}</StyledActivityName>
       {isFlow && <FlowChip size="small" />}
+      <ActivityStatusChip status={status} />
```

Comment:
The notes to engineering on the ticket says:

> Please wrap this in a new feature flag. the  enableActivityAssign feature flag will be disabled once Assign R1 is ready.

I'm not sure if this is accurate, but if it is then we'd need a new feature flag here. We should clarify that and have it removed from the description if it's not applicable

Reply:
Ah, looking more closely I think this work was originally intended for MI Assign R2 – but it got pulled into R1. So it will assume the same `enableActivityAssign` feature flag as the rest of the work I've been doing.